README.md

```diff
@@ -14,7 +14,7 @@ This project consists of 3 artifacts:
 -  [*google-auth-library-credentials*](#google-auth-library-credentials): contains base classes and
 interfaces for Google credentials
 -  [*google-auth-library-appengine*](#google-auth-library-appengine): contains App Engine
-credentials. This artifacts depends on the App Engine SDK
+credentials. This artifact depends on the App Engine SDK,
```

Comment:
The comma seems out of place here. Perhaps remove the `:` from each entry and turn each bullet point into a complete sentence?

Reply:
typo, should have been a period